lazybytez/jojo-discord-bot
# Add bot audit log configuration

## Description
Add a bot audit log that allows to track any administrative actions done in context with the bots configuration and features.
This PR is part one of two, where part two will implement the bot audit log fucntionality itself.

## Related issue
Resolves #78 

## How can this be tested?
 - Check if `/jojo audit` and its subcommands full-fill their tasks
 - Check if the configured audit log channel receives audit log mesasges
 - Check if the database sucessfuly stores configuration and log data
